.github/workflows/docker-unified.yml

```diff
@@ -472,23 +488,35 @@ jobs:
         if: ${{ needs.setup.outputs.publish != 'true' }}
         with:
           image: ${{ env.DATAHUB_MCE_CONSUMER_IMAGE }}:${{ needs.setup.outputs.unique_tag }}
-      - name: Download MCE Consumer image
+      - name: Download MAE Consumer image
         uses: ishworkh/docker-image-artifact-download@v1
         if: ${{ needs.setup.outputs.publish != 'true' }}
         with:
           image: ${{ env.DATAHUB_MAE_CONSUMER_IMAGE }}:${{ needs.setup.outputs.unique_tag }}
+      - name: check background image download status
+        run: |
+          jobs
+      - name: run quickstart
+        env:
+          DATAHUB_TELEMETRY_ENABLED: false
+          DATAHUB_VERSION: ${{ needs.setup.outputs.unique_tag }}
+        run: |
+          ./smoke-test/run-quickstart.sh
```

Comment:
I merged David's PR in which has a new Gradle task to launch quickstart, it compiles and generates docker images much faster now, should we use that?
See https://github.com/datahub-project/datahub/pull/6744

Reply:
Let's do a follow up PR for that people feel these are still slow.

Reply:
In general not a fan of big PRs. Harder for everyone to review. So let's do a follow up if required.